src/pkgdev/tatt/template.sh.jinja

```diff
@@ -0,0 +1,123 @@
+{#
+Copyright (C) 2010-2022 Gentoo tatt project
+https://gitweb.gentoo.org/proj/tatt.git/
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+#}
+#!/bin/bash
+
+main() {
+    trap "echo 'signal captured, exiting the entire script...'; exit" SIGHUP SIGINT SIGTERM
+    echo -e "USE tests started on $(date)\n" >> "{{ report_file }}"
+
+    local test_ret=0
+
+    {% for atom, is_test, use_flags in jobs %}
+    {% if is_test %}
+    {{ use_flags }} tatt_test_pkg --test {{ atom }} || test_ret=1
+    {% else %}
+    {{ use_flags }} tatt_test_pkg {{ atom }} || test_ret=1
+    {% endif %}
+    {% endfor %}
+
+    exit ${test_ret}
+}
+
+cleanup() {
+    echo "Cleaning up"
+    {% for file in cleanup_files %}
+    rm -v -f {{ file }}
```

Comment:
quotation/protection against IFS is wise here.  It's not like someone is going to shove a whitespace character in, but when they do, this shouldn't nuke errant things. :)